README.md

```diff
@@ -75,6 +75,34 @@ $ rails db:migrate
 
 The above command also ensures that an index is created on `stash_id`.
 
+## Configuration
```

Comment:
I think that there's a lot of room for improvement here, but this at least illustrates the various options for config.

Ideally I think we'd have a recommended way to handle config in each environment, or even an example app with the intended usage.

Reply:
@coderdan mentioned that a potential use-case for Rails config/credentials is in CI or production environments where users might not have a profile or JSON config files.

I was testing out handling all ActiveStash config in a credentials file, and I noticed that StashRB will raise an exception if a default profile doesn't exist. I think we'll probably want to allow StashRB to work without a JSON config file @mpalmer.

To work around this while testing, I created an "empty" profile without any actual config (my profile-config.json file contained an empty object).

Reply:
Can you show me how `CipherStash::Client.new` was being called, that causing it to raise?  It is intended to be useable without a profile at all, as long as device code auth isn't being used, so I'd like to see how you're making it explode, so I can fix either the docs or the implementation.

Reply:
> as long as device code auth isn't being used

Hmm this may have been the issue. I'll give it another try on Monday.

Reply:
Looked back over the code in StashRB and the error that I ran into on Friday. I think the issue was that I attempted to test not having a local profile by deleting my `profile-config.json` in my default profile dir, but I still had my default profile specified as my default in `~/.cipherstash/config.json`.

I'll still try testing out these config changes using access key auth today, though.